packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js

```diff
@@ -43,7 +43,7 @@ export const useBreadcrumbs = props => {
         nextFetchPolicy: 'cache-first'
     });
 
-    const categoryUrlSuffix = (data && data.category.url_suffix) || '';
```

Comment:
Wait, did we already fix this? `|| ''` means it'll use an empty string even if the configured suffix is `''`...